### PR TITLE
feat: auto-update changelog files

### PR DESCRIPTION
**Pull request by Intevel**
Related to #13 
-
This PR should not be merged until #23 is fixed. 
With the `appendFile` flag the changelog is appended to the start of the file

### CODE REVIEW

**Comment by pi0** [comment]
Thanks for this PR 💯 One small thing, we need to kinda parse the readme to add new version to the top of the readme (before first `##`). See this example changelog: https://github.com/unjs/changelogen/blob/main/CHANGELOG.md

File: package.json [hunk 1]
```diff
@@ -34,6 +34,7 @@
     "c12": "^0.2.7",
     "consola": "^2.15.3",
     "execa": "^6.1.0",
+    "fs": "^0.0.1-security",
```
Comment (pi0) [inline package.json:37]: i think this is extra

Comment (Intevel) [reply package.json:37]: What do you mean with extra?

Comment (Intevel) [reply package.json:37]: Ah thank you.

**Comment by Intevel** [comment]
> Thanks for this PR 💯 One small thing, we need to kinda parse the readme to add new version to the top of the readme (before first ##). See this example changelog:

Do you think we should read the new version from package.json and then include it in the changelog?

**Comment by pi0** [comment]
I think we can introduce the version bump feature to package.json in next iteration.

**Comment by Intevel** [comment]
Are you working on it?

**Comment by pi0** [comment]
No but plan to. With this pr, you can use `to` prop as version. It works on versioned HEAD.

**Comment by Intevel** [comment]
@pi0 Any updates on the version bump feature?

**Comment by pi0** [comment]
Hi @Intevel. Sorry it took long to be back on this. I've pushed few changes to this feature and preparing for next major version. We by default will update `CHANGELOG.md` and (preprend) the latest version in entry. I've also added new option of `--output` to specify changelog file. Using `changelogen --no-output` it will do as before behavior.

Until supporting `--bump`, the title entry is `config.to` which is latest ref.

**Pull request merged**